ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -176,7 +179,8 @@ public void close() {
           (ec, sc) -> InjectorChain.of(
               new DefaultSchemaInjector(
                   new SchemaRegistryTopicSchemaSupplier(sc.getSchemaRegistryClient())),
-              new DefaultTopicInjector(ec)
+              new TopicCreateInjector(ec),
```

Comment:
side note: Would be good to find a way for this injector chain to be _guarenteed_ to be the same as that used by interactive mode, i.e. the same code.   If it differs, this test becomes invalid.

Reply:
🤔 good point - I have some ideas but they involve code jitters. I'll flag this as a follow-up!

Reply:
Yeah, I didn't mean in this PR, just one to think about.